Akka.Serialization.MessagePack/MsgPackSerializer.cs

```diff
@@ -148,7 +172,7 @@ public override object FromBinary(byte[] bytes, Type type)
             return MessagePackSerializer.Deserialize(type, bytes,_serializerOptions);
         }
 
-        public override int Identifier => 150;
+        public override int Identifier => 151;
```

Comment:
There's an interesting question as to whether we want this to have a new serialization identifier or not.

Yes, it is now backwards compat with old version, but there's enough other stuff going on I wasn't sure whether it would be better for folks to have a new ID and just override the old serializer ID if they really want backward compat.

open to ideas here.

Reply:
I think this is a good call IMHO - better to make the new serializer opt-in; it's probably pretty low risk given that the audience using it has been small.